src/OpenTK/Platform/Linux/LinuxNativeWindow.cs

```diff
@@ -374,9 +374,10 @@ public override Point PointToScreen(Point point)
             return new Point(point.X + client.X, point.Y + client.Y);
         }
 
-        public override void ConfineCursor (bool confine)
+        public override bool CursorGrabbed
         {
-            throw new NotImplementedException ();
+            set {}
```

Comment:
Add a check that if "value == false" raise NotImplementedException. If "value == true" can just do nothing.